plugin/src/main/java/org/opensearch/sql/plugin/rest/RestPPLQueryAction.java

```diff
@@ -145,7 +151,7 @@ public void onFailure(Exception e) {
                     Metrics.getInstance()
                         .getNumericalMetric(MetricName.PPL_FAILED_REQ_COUNT_SYS)
                         .increment();
-                    reportError(channel, e, SERVICE_UNAVAILABLE);
+                    reportError(channel, e, INTERNAL_SERVER_ERROR);
```

Comment:
Does SQL needs all changes in this file?

Reply:
We support datasources in PPL, so SQL didnt come into my mind.
Probably this line change is required. I couldn't find similar structure, but will try to incorportate these changes into sql.

Reply:
This change is unnecessary for the immeditate requirements, so reverted this status code change.